sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -93,6 +93,19 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
     NOTE: If you are using WSL, do not run these commands from the Windows system drive (e.g. `/mnt/c/`).
 
+    ### Certificates - Important to know
+
+    For sample purposes, Embedded C SDK samples leverage certificates obtained when executing either `New-ArduinoZipLibrary.ps1` or `generate_arduino_zip_library.sh`. 
```

Comment:
```suggestion
    The Embedded C SDK samples leverage certificates obtained when executing either `New-ArduinoZipLibrary.ps1` or `generate_arduino_zip_library.sh`. These samples also use the Baltimore root CA to validate IoT Hub and Device Provisioning Service (DPS) server certificates.
```

Reply:
I think the message here should be different.
The idea is that the application shall use the certificates for TLS negotiation (server cert validation); 
the scripts being related to it is secondary, they are there just to support the main goal above.

My suggestion:

### Certificates - Important to know
 
The Azure IoT Hub certificates presented during TLS negotiation shall be always validated using the appropriate root CA certificate(s).
For the Node MCU ESP8266 sample our script `generate_arduino_zip_library.sh` automatically downloads the root certificate used in the United States regions (Baltimore CA certificate) and add it to the sketch project.
For other regions (and private cloud environments), please use the appropriate root CA certificate.